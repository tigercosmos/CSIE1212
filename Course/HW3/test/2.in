5
O...O
..O..
..OXX
..X..
O.X.O

X...X
.....
.OOO.
.OOO.
X...X

X...X
..O..
.OOO.
.OO..
X...X

X...X
.....
.OO..
.OOOO
X...X

.....
.XOX.
.OOOO
.XOX.
.....