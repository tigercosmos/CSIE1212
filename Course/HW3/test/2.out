O win
Draw
X win
Draw
X win
